src/Workleap.Extensions.Http.Authentication.ClientCredentialsGrant/BackchannelRetryHttpMessageHandler.cs

```diff
@@ -21,7 +22,7 @@ public BackchannelRetryHttpMessageHandler()
 
     // Microsoft documentation about resilient HTTP requests and production-grade retry jitter strategy:
     // https://learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly#add-a-jitter-strategy-to-the-retry-policy
-    private static IAsyncPolicy<HttpResponseMessage> GetRetryPolicy()
+    private static AsyncRetryPolicy<HttpResponseMessage> GetRetryPolicy()
```

Comment:
.NET 8 SDK suggested this improvement with a new roslyn analyzer warning

![image](https://github.com/gsoft-inc/wl-authentication-clientcredentialsgrant/assets/14242083/d0ee9285-d55c-4e02-a14f-fa034c709b7a)